Watchman.Discord/Areas/Responses/Services/ResponsesService.cs

```diff
@@ -28,7 +28,9 @@ public async Task<Response> GetResponseByOnEvent(string onEvent, ulong serverId
 
         public async Task AddResponse(string onEvent, string message, ulong serverId = 0)
         {
-            var addResponse = new Response(onEvent, message, serverId);
+            var query = new GetResponseQuery(onEvent);
+            var queryResult = await this._queryBus.ExecuteAsync(query);
```

Comment:
var responses = (await this._queryBus.ExecuteAsync(query)).Responses

Reply:
Faktycznie, można to skrócić, dzięki!

Reply:
Tylko jak response**s** skoro pobieram tylko jedną z domyślnych?

Reply:
to .Response 
Marcin pewnie po prostu nie zauważył że jest GetResponseQuery

Reply:
załatwione